ipatests/test_integration/test_installation.py

```diff
@@ -824,6 +824,15 @@ def test_file_permissions(self):
             msg = "rpm -V found group issues for the following files: {}"
             assert group_warnings == [], msg.format(group_warnings)
 
+    def test_ds_disable_upgrade_hash(self):
```

Comment:
It would be nice to have some lines describing test and pagure issue in doc string.